packages/patrol_cli/lib/src/runner/patrol_command.dart

```diff
@@ -146,4 +149,22 @@ abstract class PatrolCommand extends Command<int> {
   List<String> stringsArg(String name) {
     return argResults![name]! as List<String>? ?? <String>[];
   }
+
+  BuildMode get buildMode {
+    if (!_usesBuildOption) {
+      throwToolExit('This command does not support build mode option');
```

Comment:
Do you explain that `debug`, `profile` and `release` are actually _build modes_? If not, this message may still be ambiguous

Reply:
This is a classic example of "should never happen".

Thanks for spotting this. I added the following comment:

> // If this happens, it's a developer fault, not the user's.

I guess the future me will thank the present me :D